stripe/src/main/java/com/stripe/android/model/BankAccount.java

```diff
@@ -143,4 +155,51 @@ public String getLast4() {
     public String getRoutingNumber() {
         return mRoutingNumber;
     }
+
+    /**
+     * Converts a String value into the appropriate {@link BankAccountType}.
+     *
+     * @param possibleAccountType a String that might match a {@link BankAccountType} or be empty.
+     * @return {@code null} if the input is blank or of unknown type, else the appropriate
+     *         {@link BankAccountType}.
+     */
+    @Nullable
+    @BankAccountType
+    public static String asBankAccountType(@Nullable String possibleAccountType) {
+        if (BankAccount.TYPE_COMPANY.equals(possibleAccountType)) {
+            return BankAccount.TYPE_COMPANY;
+        } else if (BankAccount.TYPE_INDIVIDUAL.equals(possibleAccountType)) {
+            return BankAccount.TYPE_INDIVIDUAL;
+        }
```

Comment:
just fyi on iOS, we end up being consistently lenient on these mappings and always make it case insensitive